source/upgrade/_sysvm_templates_pre45.rst

```diff
@@ -18,8 +18,9 @@
 Update System-VM templates
 --------------------------
 
-.. note::
-   Upgrading pre-4.3 to 4.4.1 require 2 systemvm templates downloaded: the 4.3 and 4.4.
+.. warning::
+   Upgrading from 4.4 or older to 4.6.0 require 2 systemvm templates to be
```

Comment:
You can remove this line, you just need the 4.6 version (as there is no change in 4.7)